.github/workflows/simpletest-db-config.yml

```diff
@@ -0,0 +1,41 @@
+name: SimpleTest (Database Config)
+on:
+  pull_request:
+    branches: [1.*]
+jobs:
+  simpletest-db-config:
+    name: SimpleTest
+    runs-on: ubuntu-22.04
+    # Skip this job if the PR title contains "[skip tests]".
+    if: ${{ !contains(github.event.pull_request.title, '[skip tests]') }}
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['8.3']
+        database-versions: ['mariadb-10.3']
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up webserver
+        uses: ./.github/actions/set-up-webserver
+        with:
+          php-version: ${{ matrix.php-versions }}
+          database-version: ${{ matrix.database-versions }}
+
+      - name: Install Backdrop
+        run: |
+          cp ./.github/misc/settings.local.php .
+
+          # Set the config storage class to use the database.
+          echo "\$settings['config_active_class'] = 'ConfigDatabaseStorage';" >> settings.local.php
+          echo "\$config_directories['active'] = NULL;" >> settings.local.php
```

Comment:
Is that line actually needed? I didn't set that at all in my local test.

Reply:
Yeah I don't know if it's more or less confusing. I sort of like having it all in this file rather than the settings.local.php containing additional if statements.